src/main/scala/com/datastax/sparkstress/SparkCassandraStress.scala

```diff
@@ -24,7 +24,7 @@ object SaveMethod extends Enumeration {
 case class Config(
   //Test Options
   seed: Long = System.currentTimeMillis(),
-  testName: String ="WriteShortRow",
+  testName: String ="writeshortrow",
```

Comment:
not sure this is needed since we change everything to lowercase anyways, but doesn't hurt

Reply:
This is required (I hit this as a bug) because the lowercasing is done to user passed args, not to the parameter. That means the default is left untouched.